### PR TITLE
DMC DeLorean (combined)

### LINKED ISSUE #824: DMC DeLorean
**Topic:** The [DMC DeLorean](https://github.com/juliadenham/Summit_knowledge/blob/main/DMC_DeLorean.md) is a rear-engine two-passenger sports car manufactured and marketed by John DeLorean's DeLorean Motor Company.

**Folder Location:** taxonomy/knowledge/cars/delorean

**Downloads:** 
- qna.yaml: [DMC_delorean.txt](https://github.com/instructlab/taxonomy/files/15226908/DMC_delorean.txt)
- [attribution.txt](https://github.com/instructlab/taxonomy/files/15226113/delorean-attribution.txt) 

**Wiki link:**  https://en.wikipedia.org/wiki/DMC_DeLorean

### PR DESCRIPTION
**Pull request by ckadner**
**Describe the contribution to the taxonomy**

Combined knowledge submissions for DMC DeLorean from Red Hat Summit 2024.

Closes #887
Closes #889
Closes #890
Closes #933
Closes #938
Closes #949
Closes #1003
Closes #1006
Closes #1022
Closes #1050

Resolves #824

FYI @SEHANTA, @arxspecialist, @cpgill1, @jessygluekler, @joebelford, @mguginf, @nsthompson, @redhatconfuser1, @zsplaingard, @mattbrown1477

**Contribution checklist**

- [ ] The contribution was tested with `ilab generate`
- [ ] No errors or warnings were produced by `ilab generate`
- [x] All [commits are signed off](https://github.com/instructlab/taxonomy/blob/main/CONTRIBUTING.md#legal) (DCO)
- [x] The `qna.yaml` file contains at least 5 `seed_examples`
- [x] The `qna.yaml` file was [linted](https://yamllint.com) and [prettified](https://onlineyamltools.com/prettify-yaml) ([yaml-validator](https://jsonformatter.org/yaml-validator) can do both)
- [x] An `attribution.txt` file in the same folder as the `qna.yaml` file
- [x] Content does not include PII or otherwise sensitive or confidential information
- [x] Content does not include anything documented in the project's [Avoid these Topics](https://github.com/instructlab/taxonomy/blob/main/docs/SKILLS_GUIDE.md#avoid-these-topics) guidelines

### CODE REVIEW

**Comment by ckadner** [comment]
@juliadenham @kelbrown20 -- [similar to Prince Leopold Island](https://github.com/instructlab/taxonomy/pull/1154#issuecomment-2146437322), there are some key numerical values missing from the Markdown, like the miles per hour figures below

---

https://en.wikipedia.org/wiki/DMC_DeLorean#Performance

> DMC's comparison literature noted that the DeLorean could achieve **0–60 miles per hour (0–97 km/h)** in 8.8 seconds when equipped with a manual transmission, but other sources indicate an acceleration time of 9.5 seconds. When equipped with a manual transmission, the DeLorean accelerated **from 0 to 60 miles per hour (0 to 97 km/h)** in 10.5 seconds as tested by Road & Track magazine. Top speed was **claimed at 130 mph (209 km/h)**, but was tested to be a **lacking 110 mph (177 km/h)**. [Car & Driver](https://en.wikipedia.org/wiki/Car_%26_Driver) put the top speed of a 5-speed manual **at 117 mph (188 km/h) (indicated)**

https://github.com/juliadenham/Summit_knowledge/blob/main/DMC_DeLorean.md#performance

> DMC's comparison literature noted that the DeLorean could achieve in 8.8 seconds when equipped with a manual transmission, but other sources indicate an acceleration time of 9.5 seconds. When equipped with a manual transmission, the DeLorean accelerated **from** in 10.5 seconds as tested by Road & Track magazine. Top speed was **claimed at** , but was tested to be a **lacking .** [Car & Driver](https://github.com/juliadenham/Summit_knowledge/blob/main/Car_&_Driver) put the top speed of a 5-speed manual **at (indicated).**

===

https://en.wikipedia.org/wiki/DMC_DeLorean#Base_price

> Upon release in 1981, a DeLorean had a base MSRP of $25,000, or equivalent to $84,000 in 2023. MSRP increased in 1982 to $29,825,[[57]](https://en.wikipedia.org/wiki/DMC_DeLorean#cite_note-59) equivalent to $94,000 in 2023, and again in 1983 to $34,000,[[58]](https://en.wikipedia.org/wiki/DMC_DeLorean#cite_note-60) equivalent to $104,000 in 2023.

https://github.com/juliadenham/Summit_knowledge/blob/main/DMC_DeLorean.md#base-price

> Upon release in 1981, a DeLorean had a base MSRP of $25,000, or . MSRP increased in 1982 to $29,825, , and again in 1983 to $34,000, .





---


Should we update the Markdown files or remove all questions that are referring to missing numbers?

File: knowledge/cars/delorean/qna.yaml [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+- answer: "DMC's comparison literature noted that the DeLorean could achieve 0\u2013\
+    60 miles per hour (0\u201397 km/h) in 8.8 seconds when equipped with a manual\
+    \ transmission, but other sources indicate an acceleration time of 9.5 seconds.\
+    \ When equipped with a manual transmission, the DeLorean accelerated from\
+    \ 0 to 60 miles per hour (0 to 97 km/h) in 10.5 seconds as tested by Road\
+    \ & Track magazine. Top speed was claimed at 130 mph (209 km/h), but was tested\
+    \ to be a lacking 110 mph (177 km/h). Car & Driver put the top speed of a\
+    \ 5-speed manual at 117 mph (188 km/h) (indicated). The car was described\
+    \ as \"not quick for a sports/GT car in this price category\" by Road & Track."
+  question: What is the performance of the DMC DeLorean?
```
Comment (ckadner) [inline knowledge/cars/delorean/qna.yaml:42]: The speed and acceleration figures here are from Wikipedia but are not present in the Markdown

https://github.com/instructlab/taxonomy/pull/1156#issuecomment-2146928464

File: knowledge/cars/delorean/qna.yaml [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+    \ When equipped with a manual transmission, the DeLorean accelerated from\
+    \ 0 to 60 miles per hour (0 to 97 km/h) in 10.5 seconds as tested by Road\
+    \ & Track magazine."
+  question: How long did it take for the DeLorean to achieve 60 miles-per-hour?
+- answer: '6'
+  question: How many DeLoreans were used in the "Back to the Future movies?"
+- answer: '3'
+  question: How many golden DeLoreans were ever made?
+- answer: Upon release in 1981, a DeLorean had a base MSRP of $25,000, or equivalent
+    to $84,000 in 2023. MSRP increased in 1982 to $29,825, equivalent to $94,000
```
Comment (ckadner) [inline knowledge/cars/delorean/qna.yaml:55]: The dollar figures here are from Wikipedia but are not all present in the Markdown

https://github.com/instructlab/taxonomy/pull/1156#issuecomment-2146928464

File: knowledge/cars/delorean/qna.yaml [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+    and telescopic shock absorbers. The front suspension uses double wishbones,
+    while the rear is a multi-link setup.
+  question: What kind of suspension did the DeLorean have?
+- answer: Stainless-steel panels were fixed to a fiberglass underbody. The
+    underbody was affixed to a steel backbone chassis with Y-frames at either
+    end, which was derived from the Lotus Esprit platform. The chassis was
+    coated with epoxy, a material to protect steel against corrosion.
+  question: What materials was the Delorean body made from?
+- answer: Upon release in 1981, a DeLorean had a base MSRP of $25,000, or equivalent
+    to $84,000 in 2023. MSRP increased in 1982 to $29,825, equivalent to $94,000
```
Comment (ckadner) [inline knowledge/cars/delorean/qna.yaml:105]: The dollar figures here are from Wikipedia but are not all present in the Markdown

https://github.com/instructlab/taxonomy/pull/1156#issuecomment-2146928464

**Comment by juliadenham** [comment]
Hi Christian, I'm not sure how that happened in the markdown file but can fix it and add a new commit hash in the spreadsheet. Thanks for catching that

**Comment by ckadner** [comment]
https://github.com/instructlab/taxonomy/issues/1160

**Comment by mingxzhao** [comment]
@instructlab-bot precheck

**Comment by mingxzhao** [comment]
@instructlab-bot generate

File: knowledge/cars/delorean/qna.yaml (hunk 1 above)
Comment (ckadner) [reply knowledge/cars/delorean/qna.yaml:42]: @juliadenham updated the Markdown in commit 0feadeec8794767f2ea89b6970ba78936bd1f505

**Comment by jjasghar** [comment]
@instructlab-bot precheck

**Comment by jjasghar** [comment]
Hi! :wave:
With the new update and release of 0.18.0, our [base knowledge submissions](https://github.com/instructlab/taxonomy/?tab=readme-ov-file#knowledge-yaml-examples) have drastically changed. This has given us an opportunity to significantly improve the accuracy of our fine-tuned models, which is a major win for our community!

But alas, that has caused some changes that this PR isn't able to continue forward. We greatly appreciate the work you've put into this, and wish we could accept it but the amount of work to get it to be `version: 3` is unrealistic for us to ask.
If you _do_ want to do this, please don't hesitate to take the `qna.yaml` update it, and build it out. Otherwise, we hope you come back to our community with a new submission and help build out our Open Source community-driven LLM.

If you have any questions directly, please don't hesitate to comment on the PR or email me at [awesomeATinstructlab.ai](http://awesomeatinstructlab.ai/) (yes, really), and I'll do my best to answer your questions.

**Pull request closed without merging**